frontend/src/metabase/components/EditWarning.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+class EditWarning extends Component {
```

Comment:
FYI if a component doesn't have state or use lifecycle methods we usually make it a "stateless function component" like

```
function EditWarning({ title }) {
  // ...
}
EditWarning.propTypes = {
  // ...
}
```